tests/Avalonia.Benchmarks/Layout/CalendarBenchmark.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using Avalonia.Controls;
+using Avalonia.UnitTests;
+using BenchmarkDotNet.Attributes;
+
+namespace Avalonia.Benchmarks.Layout
+{
+    [MemoryDiagnoser, InProcess, IterationCount(16)]
+    public class CalendarBenchmark : IDisposable
+    {
+        private readonly IDisposable _app;
+        private readonly TestRoot _root;
+        
+        public CalendarBenchmark()
+        {
+            _app = UnitTestApplication.Start(TestServices.StyledWindow.With(renderInterface: new NullRenderingPlatform(), threadingInterface: new NullThreadingPlatform()));
```

Comment:
Tiny nit: this line is too long for the GitHub source view meaning scrolling is needed.